.gitignore

```diff
@@ -1,4 +1,7 @@
 /*.csv#
 *.badges
 *.swp
-/*.pyc
\ No newline at end of file
+/*.pyc
```

Comment:
`/*.pyc` ignores all pyc files in the root directory. `*.pyc` ignores all .pyc files in all dorectories. 
Github Pyhton ignore file: https://github.com/github/gitignore/blob/master/Python.gitignore

Reply:
Please remove this file: https://github.com/fossasia/badgeyay/pull/86/files#diff-60cfe20702be862f7a97b99fea0b072a

Reply:
@niccokunzmann Sorry I didnt get it